### PR TITLE
export the logs of Volcano components when the e2e test workflow fails

### LINKED ISSUE #3429: export the logs of Volcano components when the e2e test workflow fails
#### What would you like to be added:
print the logs of volcano components in clean up stage
https://github.com/volcano-sh/volcano/blob/ab2def642f6e5e594b9fbdf0176f3426270826ed/hack/run-e2e-kind.sh#L75-L86
, the logs of volcano components are generated by
https://github.com/volcano-sh/volcano/blob/ab2def642f6e5e594b9fbdf0176f3426270826ed/hack/run-e2e-kind.sh#L68-L73
when e2e test is failed
https://github.com/volcano-sh/volcano/blob/ab2def642f6e5e594b9fbdf0176f3426270826ed/hack/run-e2e-kind.sh#L160-L163

#### Why is this needed:

when the e2e test is failed, we need those logs for troube shorting

### PR DESCRIPTION
**Pull request by panoswoo**
Resolves #3429

### CODE REVIEW

File: hack/run-e2e-kind.sh [hunk 1]
```diff
@@ -72,6 +72,30 @@ function generate-log {
     kubectl logs deployment/${CLUSTER_NAME}-scheduler -n ${NAMESPACE} > volcano-scheduler.log
 }
 
+function show-log() {
+  if [ -f volcano-admission.log ]; then
```
Comment (calvin0327) [inline hack/run-e2e-kind.sh:76]: the logic of printing volcano component is similar, whether it's better to do it with a loop.

Comment (panoswoo) [reply hack/run-e2e-kind.sh]: > the logic of printing volcano component is similar, whether it's better to do it with a loop.

good idea

**Review by lowang-bh — APPROVED** [review]

**Review by hwdef — COMMENTED** [review]
/lgtm

**Review by lowang-bh — APPROVED** [review]

**Comment by lowang-bh** [comment]
Hi, @william-wang , please merge this pr with high-priority. thanks. cc @Monokaix

File: hack/run-e2e-kind.sh [hunk 2]
```diff
@@ -72,6 +72,19 @@ function generate-log {
     kubectl logs deployment/${CLUSTER_NAME}-scheduler -n ${NAMESPACE} > volcano-scheduler.log
 }
 
+function show-log() {
+  log_files=("volcano-admission.log" "volcano-controller.log" "volcano-scheduler.log")
+  for log_file in "${log_files[@]}"; do
+    if [ -f "$log_file" ]; then
+      echo "Showing ${log_file}..."
+      cat "$log_file"
```
Comment (Monokaix) [inline hack/run-e2e-kind.sh:80]: This seems that will print lots of logs.

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**